packages/ember-data/lib/system/snapshot.js

```diff
@@ -12,7 +12,7 @@ var get = Ember.get;
   @constructor
   @param {DS.Model} internalModel The model to create a snapshot from
 */
-export default function Snapshot(internalModel) {
```

Comment:
Nah, if that is the style in the codebase already it is better to be consistent (or update them all).